prime/src/data/TokenData.ts

```diff
@@ -1,83 +1,24 @@
 import { Address, chain } from 'wagmi';
 
 import {
-  UsdcLogo,
-  WbtcLogo,
-  WethLogo,
-  VeloLogo,
   DaiLogo,
-  OpLogo,
-  WstEthLogo,
   FraxLogo,
-  UniLogo,
+  GmxLogo,
   LyraLogo,
+  MagicLogo,
+  MimLogo,
+  OpLogo,
   PerpLogo,
+  UniLogo,
+  UsdcLogo,
+  UsdtLogo,
+  VeloLogo,
+  WbtcLogo,
+  WethLogo,
+  WstEthLogo,
 } from '../assets/svg/tokens';
 import { Token } from './Token';
 
-const USDC_OPTIMISM = new Token(
-  chain.optimism.id,
-  '0x7f5c764cbc14f9669b88837ca1490cca17c31607',
-  6,
-  'USDC',
-  'USD Coin',
-  UsdcLogo
-);
-
-const WETH_OPTIMISM = new Token(
-  chain.optimism.id,
-  '0x4200000000000000000000000000000000000006',
-  18,
-  'WETH',
-  'Wrapped Ether',
-  WethLogo
-);
```

Comment:
I prefer organizing them by chain rather than token, so that when we add new chains all the changes show up in one git diff block.

Reply:
I just copy and pasted this from earn